katago_server/games/models/training_game.py

```diff
@@ -15,12 +15,12 @@
 training_data_storage = FileSystemStorage(location="/data/training_npz", base_url="/media/training_npz/")
 validate_zip = FileValidator(max_size=1024 * 500, content_types=["application/zip"])
 
-def validate_game_npzdata(game):
-    npz_file = game.training_data_file
+def validate_game_npzdata(training_data_file,run):
```

Comment:
nits: space between `file,` and `run`

Reply:
I am not very strict on python style, very likely in all the changes I made before this there are also many more little things like this. If it bothers you, then I'd be fine with you submitting a pull request to fix up the style again, or else you if you could let me know what automated tool should be run to styleify the whole repo, that would help.